src/renderer/components/cluster-manager/cluster-manager.tsx

```diff
@@ -1,52 +1,116 @@
 import "./cluster-manager.scss"
 import React from "react";
-import { computed } from "mobx";
-import { observer } from "mobx-react";
-import { App } from "../app";
+import { WebviewTag } from "electron";
+import { Redirect, Route, Switch } from "react-router";
+import { observable, reaction } from "mobx";
+import { disposeOnUnmount, observer } from "mobx-react";
+import { clusterIpc } from "../../../common/cluster-ipc";
+import { cssNames } from "../../utils";
 import { ClustersMenu } from "./clusters-menu";
 import { BottomBar } from "./bottom-bar";
-import { cssNames, IClassName } from "../../utils";
-import { Terminal } from "../dock/terminal";
-import { i18nStore } from "../../i18n";
-import { themeStore } from "../../theme.store";
-import { clusterStore, getHostedClusterId, isNoClustersView } from "../../../common/cluster-store";
-import { CubeSpinner } from "../spinner";
-
-interface Props {
-  className?: IClassName;
-  contentClass?: IClassName;
+import { LandingPage, landingRoute, landingURL } from "../+landing-page";
+import { Preferences, preferencesRoute } from "../+preferences";
+import { Workspaces, workspacesRoute } from "../+workspaces";
+import { AddCluster, addClusterRoute } from "../+add-cluster";
+import { ClusterView } from "./cluster-view";
+import { clusterViewRoute, clusterViewURL, getMatchedCluster, getMatchedClusterId } from "./cluster-view.route";
+import { ClusterId, clusterStore } from "../../../common/cluster-store";
+import logger from "../../../main/logger";
+
+interface LensView {
+  isLoaded?: boolean
+  clusterId: ClusterId;
+  view: WebviewTag
+}
+
+const lensViews = observable.map<ClusterId, LensView>();
+
+export function hasLoadedView(clusterId: ClusterId): boolean {
+  return !!lensViews.get(clusterId)?.isLoaded;
+}
+
+// todo: figure out how to replace <webview>-tag to <iframe> with nodeIntegration=true
```

Comment:
Maybe or https://github.com/react-native-community/react-native-webview

Reply:
it was more question "why <iframe> previously get to work with vue-app" 
p.s your link follows to `react-native-community`